content/articles/responsive_flutter_design_using_mediaquery_class/index.md

```diff
@@ -0,0 +1,676 @@
+# Responsive flutter design using MediaQuery Class
+
+
+
+In this tutorial, you will acquire the knowledge of how to set up your flutter application to be responsive and resolve well on all devices.
+
```

Comment:
Change to " This tutorial will teach us how to set up a flutter application to be responsive and resolve satisfactorily on all devices."